ls1intum/Artemis
# `General`: Make spacings consistent

### Checklist
#### General


#### Client

### Motivation and Context


- Makes the UI consistent with bootstrap configuration leading to better ease of use.
- Improves accessibility for e.g. better screen reader support.


### Description

- Changes some margins and padding to be consistent with our bootstrap configuration
- Simplifies scss so that future changes are easier
- Improves DOM to use better elements and improve accessibility

### Steps for Testing

Prerequisites:
- Instructor account as some pages are only visible with those permissions

1. See the screenshots below and follow the navigation instructions.
2. Check that the screenshots match and the UI is not broken various screen sizes.

### Review Progress






### Screenshots

/course management > Exercises
![artemis-test1 artemis in tum de_course-management_66_exercises](https://user-images.githubusercontent.com/25489463/222721098-7616fd6c-e8ff-4e9c-82d3-412dd33cb14f.png)
![localhost_9000_courses_11_exercises (1)](https://user-images.githubusercontent.com/25489463/222721136-aaf033cf-ec11-4fd2-804c-b2f6d5498e52.png)

/courses > any course
![artemis-test1 artemis in tum de_course-management_66_exercises (1)](https://user-images.githubusercontent.com/25489463/222721125-c9d80b94-f73e-497e-9901-533cc0b07582.png)
![localhost_9000_course-management_9](https://user-images.githubusercontent.com/25489463/222721129-69370cdc-321c-443b-a78f-e69eb0913c75.png)

/courses > any course > Exercises
![artemis-test1 artemis in tum de_courses](https://user-images.githubusercontent.com/25489463/222721128-83c0b380-62dd-4f13-b98a-2256aa7f5154.png)
![localhost_9000_courses_11_exercises](https://user-images.githubusercontent.com/25489463/222721134-61d2023f-f4dd-4b35-aced-9471e7ab3f14.png)
